mailbox/api/src/test/java/org/apache/james/mailbox/MailboxManagerTest.java

```diff
@@ -580,31 +579,23 @@ public void searchForMessageShouldReturnMessagesFromAllMyMailboxesIfNoMailboxesA
             .getSupportedMessageCapabilities()
             .contains(MailboxManager.MessageCapabilities.UniqueID));
 
-        boolean isRecent = false;
-
         session = mailboxManager.createSystemSession(USER_1);
 
         MailboxPath cacahueteFolder = MailboxPath.forUser(USER_1, "CACAHUETE");
         MailboxId cacahueteMailboxId = mailboxManager.createMailbox(cacahueteFolder, session).get();
         MessageManager cacahueteMessageManager = mailboxManager.getMailbox(cacahueteMailboxId, session);
-        MessageId cacahueteMessageId = cacahueteMessageManager.appendMessage(
-            new ByteArrayInputStream("Subject: test\r\n\r\ntestmail".getBytes()),
-            new Date(),
-            session,
-            isRecent,
-            new Flags())
+        MessageId cacahueteMessageId = cacahueteMessageManager.appendMessage(MessageManager.AppendCommand.builder()
```

Comment:
we should call recent(false) explicitelty

Reply:
We do not care about this information.... To not make the tests unreadable I chose not to add it.